spec/changelog.yml

```diff
@@ -2,6 +2,12 @@
 ## This file documents changes in the package specification. It is NOT a package specification file.
 ## Newer entries go at the bottom of each in-development version.
 ##
+- version: 3.2.0-next
+  changes:
+  - description: Add validation for ingest pipeline rename processor
+    type: breaking-change
+    type: enhancement
+    link: https://github.com/elastic/package-spec/pull/690
```

Comment:
As `3.1.0` has not been released yet, probably it could be moved to the `3.1.0-next` changelog entry, couldn't it? Not sure if I could be missing something here.

Reply:
Since v3 we are not strictly following semver for the package spec, I tried to explain this in https://github.com/elastic/package-spec#changing-a-specification. The summary would be that we only must use new minors for changes that require changes in the stack.

The reason for this change in v3 is that we are using this version for the compatibility of packages with the stack (since https://github.com/elastic/ingest-dev/issues/2185, internal only sorry), so we have to see if it is breaking specially regarding the stack, and not the packages themselves.

> The difference is that this change is breaking, while the one under `3.1.0-next` isn't. The PR for it has been merged, so maybe we could release 3.1.0?

The change about `subobjects` is in 3.1.0 because it needs support in the stack, even if it is indeed not breaking for packages. Before releasing 3.1.0 we need to be sure that this support has been implemented, and we will also need to update the max version supported by Kibana offerings ([these settings](https://github.com/elastic/kibana/blob/15f999fbdb2322f9ac6c2318c753bc502ee1d2ed/config/serverless.oblt.yml#L44)).

----

This change adds a new validation to fix issues in packages, and doesn't require changes in the stack, so it can go in the next patch version, `3.0.5-next` in this case. If we wanted to release it on next minor, it should be in `3.1.0-next` that hasn't been released yet.